data/templates/styles.html

```diff
@@ -164,7 +164,8 @@
 code{white-space: pre-wrap;}
 span.smallcaps{font-variant: small-caps;}
 span.underline{text-decoration: underline;}
-div.column{display: inline-block; vertical-align: top; width: 50%;}
+div.columns{display: flex;}
+div.column{flex: 1;}
```

Comment:
cool, yes, the previous CSS is probably ancient...

tho thougths:

instead of doing `div.column{flex: 1;}`, could also do `div > div {flex: 1;}`, then we wouldn't have to write the `.column` class in markdown... but yeah since the `:::` syntax needs a class anyway, probably just as well to keep it like this.

while we're at it, I'd probably add to the `.columns` some kind of gap, for example setting the gap to `2em` except on small screens where that would be too much set it to 4% of the viewportwidth:

    gap: min(4vw, 2em);

Reply:
Good idea! I've added the gap, but opted for `1.5em`. I didn't know CSS had a `min` function, that's extremely useful!